### PR TITLE
 * Replace System.Resources.ReaderWriter

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Part of #199.

### CODE REVIEW

File: src/Microsoft.Extensions.Localization/project.json [hunk 1]
```diff
@@ -16,11 +16,12 @@
...
   },
   "frameworks": {
-    "net451": {},
+    "net451": { },
     "dotnet5.4": {
       "dependencies": {
         "System.Collections.Concurrent": "4.0.11-*",
-        "System.Resources.ReaderWriter": "4.0.0-*"
+        "System.Resources.Reader": "4.0.0-*",
+        "System.Resources.Writer": "4.0.0-*"
```
Comment (pranavkm) [inline src/Microsoft.Extensions.Localization/project.json]: Why do we need the writer here?

**Comment by Eilon** [comment]
:shipit:

**Comment by BrennanConroy** [comment]
Don't push until we grab newer CoreCLR bits since this uses a new package we don't have on our feed yet.

**Comment by Eilon** [comment]
@BrennanConroy yup, aware of that :smile:

**Comment by ryanbrandenburg** [comment]
@BrennanConroy ok, let me know when you guys have that done.

**Comment by KrzysztofCwalina** [comment]
BTW, we recently changed the feeds. This is where the component is located: https://www.myget.org/feed/dotnet-core/package/nuget/System.Resources.Reader

**Comment by Eilon** [comment]
@KrzysztofCwalina we don't pull directly from those feeds in this build - we always take snapshots of CoreCLR as part of a separate process. However, isn't that the same feed as it always was, though?

**Comment by BrennanConroy** [comment]
Ya, thats the feed we've always used

**Comment by KrzysztofCwalina** [comment]
We changed from dotnet-corefx to dotnet-core feed. I think the reson is that we ran out of space on the old feed ;-(

**Comment by Eilon** [comment]
@KrzysztofCwalina ah ok I don't see us using the old feed anywhere. I'm pretty sure we always used the `dotnet-core` feed (not `dotnet-corefx`).

**Pull request merged**